preview/MsixCore/MsixCoreInstaller/InstallUI.cpp

```diff
@@ -660,6 +693,23 @@ void UI::UpdateDisplayPercent(float displayPercent)
     ShowWindow(g_staticPercentText, SW_SHOW);
 }
 
+void UI::DisplayError(HRESULT hr)
+{
+    g_installing = false;
+    ShowWindow(g_percentageTextHWnd, SW_HIDE);
+    ShowWindow(g_staticPercentText, SW_HIDE);
+    ShowWindow(g_progressHWnd, SW_HIDE);
+    ShowWindow(g_checkboxHWnd, SW_HIDE);
+    ShowWindow(g_CancelbuttonHWnd, SW_HIDE);
+    
+    //Show Error Box
+    ShowWindow(g_staticErrorTextHWnd, SW_SHOW);
+    ShowWindow(g_staticErrorDescHWnd, SW_SHOW);
+    std::wstringstream errorDescription;
+    errorDescription << "App installation failed with error message: An internal error occurred with error " << hr << ".";

```

Comment:
>"App installation failed with error message: An internal error occurred with error  [](start = 24, length = 83)

i think best to make this a string resource?

Reply:
Modified to use string resource.